CONTRIBUTING.md

```diff
@@ -1,22 +1,21 @@
-# Contributing to the Metaschema Project
+# Contributing to this Project
 
-This page is for potential contributors to this project. It provides basic information on the project, describes the main ways people can make contributions, explains how to report issues relating to the project and projecta rtifacts, and lists pointers to additional sources of information.
+This page is for potential contributors to this project. It provides basic information on the project, describes the main ways people can make contributions, explains how to report issues relating to the project and project artifacts, and lists pointers to additional sources of information.
 
 ## Project approach
 
-This project uses an agile approach for development, where we focus on implementing the 20% of the functionality that solves 80% of the problem. We’re trying to focus on the core capabilities that are needed to provide the greatest amount of benefit. Because we’re working on a small set of capabilities, this allows us to make very fast progress. We’re building the features that we believe solve the biggest problems to provide the most value. We provide extension points that allow uncovered cases to be supported by others.
+This project uses an agile approach for development. We’re trying to focus on the core capabilities that are needed to provide the greatest amount of benefit. Because we’re working on a small set of capabilities, this allows us to make very fast progress. We’re building the features that we believe solve the biggest problems to provide the most value to the most people. We provide extension points that allow uncovered cases to be supported by others.
 
-We track our current work items using GitHub [project cards](../../projects). Our active project is typically the lowest numbered open project within the previously referenced page.
+We track our current work items using GitHub [project cards](../../projects).
```

Comment:
Non-blocking: re my comment about the new (not classic) [projects board](https://github.com/orgs/usnistgov/projects/44) board? This link only works because we didn't point to the specific board.

```suggestion
We track our current work items using GitHub [project cards](https://github.com/orgs/usnistgov/projects/44).
```

(Also, the board is not public, but that is orthogonal to the PR.)